rcl_action/test/rcl_action/test_graph.cpp

```diff
@@ -320,8 +316,6 @@ class TestActionGraphMultiNodeFixture : public CLASSNAME(TestActionGraphFixture,
 
     ret = rcl_shutdown(&this->remote_context);
     EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
-    ret = rcl_context_fini(&this->remote_context);
```

Comment:
I believe the changes to this file should be reverted. We should finalize the context objects in the tear down functions.

Reply:
Missed that file earlier. Reverted.